client/signup/config/flows-pure.js

```diff
@@ -129,14 +129,11 @@ export function generateFlows( {
 		},
 		{
 			name: 'link-in-bio',
-			steps: getAddOnsStep(
-				isEnabled( 'signup/professional-email-step' )
-					? [ 'user', 'domains', 'emails', 'plans' ]
-					: [ 'user', 'domains', 'plans' ]
-			),
-			destination: ( dependencies ) => getStepperFlowDestination( dependencies, 'link-in-bio' ),
+			steps: [ 'domains', 'plans' ],
+			destination: ( dependencies ) =>
+				`/setup/completingPurchase?flow=link-in-bio&siteSlug=${ dependencies.siteSlug }`,
```

Comment:
```suggestion
				`/setup/completingPurchase?flow=link-in-bio&siteSlug=${ encodeURIComponent( dependencies.siteSlug ) }`,
```

Reply:
Thank you for your review @alshakero, the 'preparing checkout' issue you pointed will be handled on the next PR.
🥇